templates/BlueGrey/index.tmpl

```diff
@@ -55,14 +55,6 @@
   [websvn-endtest]
     </table>
   [websvn-test:treeview]
-    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/group-collapse.js"></script>
-    [websvn-test:opentree]
-    [websvn-else]
-    <script type="text/javascript">
-    //<![CDATA[
-    collapseAllGroups();
-    //]]>
-    </script>
-    [websvn-endtest]
+    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/collapsible.js"></script>
```

Comment:
You can't just use the one line for `collapsible.js` only, because that file contains code which uses jQuery always. You always need both lines:

    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/jquery.min.js"></script>
    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/collapsible.js"></script>

Reply:
I have added the 
```
    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/jquery.min.1.9.1.js"></script>
    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/collapsible.js"></script>
```
in all the three index.tmpl